### PR TITLE
Use relative imports for encoded.

### PR DESCRIPTION
**Pull request by netsettler**
A step on the way to poetry. This uses relative imports rather than absolute imports for encoded.

### CODE REVIEW

File: src/encoded/tests/test_search.py [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-def delay_rerun(*args):
-    """ Rerun function for flaky """
-    time.sleep(1)
-    return True
+from snovault import TYPES, COLLECTIONS
+from snovault.elasticsearch.indexer_utils import get_namespaced_index
+from ..commands.run_upgrader_on_inserts import get_inserts
+# Use workbook fixture from BDD tests (including elasticsearch)
+from .workbook_fixtures import app_settings, app, workbook
+from ..utils import delay_rerun
```
Comment (j1z0) [inline src/encoded/tests/test_search.py]: Codacy Issue found: [Unused delay_rerun imported from utils](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5006303)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5006303&bid=16515285)

File: src/encoded/upgrade/page.py [hunk 2]
```diff
@@ -1,9 +1,8 @@
-from snovault import (
-    upgrade_step,
-)
 import json
-from encoded.schema_formats import is_uuid
+
+from snovault import upgrade_step
 from pkg_resources import resource_filename
+from ..schema_formats import is_uuid
```
Comment (j1z0) [inline src/encoded/upgrade/page.py]: Codacy Issue found: [Unused is_uuid imported from schema_formats](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5006303)

File: src/encoded/tests/test_create_mapping.py [hunk 3]
```diff
@@ -1,10 +1,14 @@
 import pytest
+
+from snovault import COLLECTIONS
 from unittest.mock import patch, MagicMock
+from ..commands.create_mapping_on_deploy import ITEM_INDEX_ORDER
+from ..commands.create_mapping_on_deploy import get_deployment_config
+# TODO: We should not be importing *. Even stranger, PyCharm says we don't use anything from there. -kmp 14-Feb-2020
```
Comment (willronchetti) [inline src/encoded/tests/test_create_mapping.py]: Does removing the below import have any side effects? Also there are still imports in `test_create_mapping` below if you want to pull those up.

File: src/encoded/tests/test_search.py [hunk 4]
```diff
@@ -1,20 +1,18 @@
...
 
 
 pytestmark = [pytest.mark.working, pytest.mark.schema, pytest.mark.indexing] #pytest.mark.flaky(rerun_filter=delay_rerun)]
```
Comment (willronchetti) [inline src/encoded/tests/test_search.py]: Might be worth adding flaky back but dropping the `delay_rerun` to maybe 10 seconds?

File: src/encoded/tests/test_create_mapping.py (hunk 3 above)
Comment (netsettler) [reply src/encoded/tests/test_create_mapping.py]: I didn't try removing it. But I think PyCharm code walks it. Variables would light up red as having no definition if it created a problem. Instead, it makes this gray, indicating it thinks it's unused. (What it doesn't codewalk is whether anyone imports anything from this file that might have been an apparently-unused import.) I put the comment as a note to remind myself to look more at this later.

File: src/encoded/tests/test_search.py (hunk 4 above)
Comment (netsettler) [reply src/encoded/tests/test_search.py]: Let me know if the change I made is what you meant.

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5006303&bid=16515285)

File: src/encoded/upgrade/page.py (hunk 2 above)
Comment (j1z0) [inline src/encoded/upgrade/page.py]: Codacy Issue found: [Unused is_uuid imported from schema_formats](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5006303)

**Pull request merged**